shared/common-adapters/popup-menu.desktop.js

```diff
@@ -51,6 +53,10 @@ class Menu extends Component<void, Props, void> {
   }
 }
 
+PopupMenu.defaultProps = {
+  visible: true,
```

Comment:
That's a good point. We currently don't handle animations but it would be necessary for them. I could update this PR so that all consumers use the visible prop so that when we do support animations they all seamlessly get them.